source/common/upstream/load_balancer_impl.h

```diff
@@ -459,6 +459,7 @@ class EdfLoadBalancerBase : public ZoneAwareLoadBalancerBase,
   const absl::optional<Runtime::Double> aggression_runtime_;
   TimeSource& time_source_;
   MonotonicTime latest_host_added_time_;
+  double min_weight_percent_;
```

Comment:
It's clear what this is for in the context of this PR, but in the load balancer file the name can be confusing. Can you change it to something that makes it clear this is specifically for the slow-start weight modifications? Something like `slow_start_min_weight_pct_`

Reply:
done